Stride-Labs/stride
# Remove address validation from query

**Summary**
UserRedemptionRecords are now keyed on receiver, which isn't a Stride address. So we shouldn't try to validate addresses as Stride addresses in this query.

**Test plan**
1. Run dockernet
2. Query `http://localhost:1317/Stride-Labs/stride/records/user_redemption_record_for_user/GAIA/10/cosmos1g6qdx6kdhpf000afvvpte7hp0vnpzapuyxp8uf/100`
3. Observe result
```
{"user_redemption_record":[{"id":"GAIA.2.cosmos1g6qdx6kdhpf000afvvpte7hp0vnpzapuyxp8uf","receiver":"cosmos1g6qdx6kdhpf000afvvpte7hp0vnpzapuyxp8uf","native_token_amount":"20020","denom":"uatom","host_zone_id":"GAIA","epoch_number":"2","claim_is_pending":false,"st_token_amount":"20000"}],"pagination":null}
```
